assemblies/pentaho-war/pom.xml

```diff
@@ -898,39 +895,6 @@
       <artifactId>hibernate-ehcache</artifactId>
       <version>${hibernate-ehcache.version}</version>
     </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-core</artifactId>
-      <version>${fasterxml.jackson.core.version}</version>
-      <exclusions>
-        <exclusion>
-          <artifactId>*</artifactId>
-          <groupId>*</groupId>
-        </exclusion>
-      </exclusions>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-annotations</artifactId>
-      <version>${fasterxml.jackson.core.version}</version>
-      <exclusions>
-        <exclusion>
-          <artifactId>*</artifactId>
-          <groupId>*</groupId>
-        </exclusion>
-      </exclusions>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-databind</artifactId>
-      <version>${fasterxml.jackson.core.version}</version>
-      <exclusions>
```

Comment:
@pentaho-lmartins Isn't this removing jackson from the provided libs of the war distribution? Similiar to my comment on the pdi client lib module. This would mean that plugins would not have access to jackson and would need to include it by their own.
Also, a general check that can be made, specially in maven modules that are "assemblies", is to do a diff on in the final artifact and check if we're removing anything that could have an undesired impact.

Reply:
@graimundo, I can confirm that `jackson-databind`, `jackson-core` and `jackson-annotations` are present in the _lib_ of the war file. Just checked by downloading and opening the CE and EE versions from the last test stack build.

All three jackson dependencies come as a transitive from **pentaho-platform-extensions**.

Ditto for the **pdi-core-libs** and **pdi-libs** assemblies. Both assemblies include kettle-engine and only exclude `javax.servlet`. So jackson dependencies are included as transitives.

Do we want to make these dependencies explicit?

Reply:
@pentaho-lmartins I think that for the scope of this story it is ok to remove the explicit dependencies and let them come in transitively. That means we are assuming that the lib contains the necessary libs for the Pentaho code base and that we are not putting anything there just for the sake of third parties. So lets go with this solution to unblock the work and create that followup thread that we talked about. If necessary, as a result of that discussion we can re add the dependencies explicitly later on.